src/app/business/services/dynamic-form/dynamic-form.component.ts

```diff
@@ -209,10 +217,87 @@ export class DynamicFormComponent implements OnInit {
         this.requestBody.user_id = this.default_parameters['user_id'];
         this.requestBody.name = formObject.instanceName;
         this.requestBody.description = formObject.instanceDescription;
-        this.requestBody.parameters = _.omit(formObject, ['instanceName', 'instanceDescription']);
+        if(this.showAnalysis){
+          let analysisParam = this.checkAnalysisParameters(formObject);
+          this.requestBody.parameters = analysisParam;
+        }else{
+          this.requestBody.parameters = _.omit(formObject, ['instanceName', 'instanceDescription']);
+        }
         this.createInstance(this.requestBody);
       }
 
+      checkAnalysisParameters(formObject){
+        if(!this.createCluster.clusterForm.valid){
+          for(let i in this.createCluster.clusterForm.controls){
+              this.createCluster.clusterForm.controls[i].markAsTouched();
+          }
+          return;
+        }
+        let requestParams = this.getRequestParam(this.createCluster.clusterForm.value,formObject);
+        return requestParams;
+      }
+      getRequestParam(value,formObject){
+        let configureValue = this.createCluster.setConfigureForm.value;
+        let actionOnFailure;
+        if(configureValue['action'] == "Cancel and wait"){
+          actionOnFailure = "CANCEL_AND_WAIT";
+        }else if(configureValue['action'] == "Terminate cluster"){
+          actionOnFailure = "TERMINATE_CLUSTER";
+        }else{
+          actionOnFailure = configureValue['action'].toUpperCase();
+        }
+        let params = {
+          ip_addr: location.hostname,
+          port: location.port,
+          name: formObject.instanceName,
+          tenant_id: this.paramStor.CURRENT_TENANT().split("|")[1],
```

Comment:
Please check with @himanshuvar . Tenant ID is being passed as part of the API call. This may not be passed as a request body param.